CHANGELOG/CHANGELOG-3.6.md

```diff
@@ -46,6 +46,7 @@ See [code changes](https://github.com/etcd-io/etcd/compare/v3.5.0...v3.6.0).
 
 ### Package `server`
 
+- Decreased [`--snapshot-count` default value from 100,000 to 10,000](https://github.com/etcd-io/etcd/pull/15408)
```

Comment:
I suggest to move it under the section "etcd server".

Reply:
Good catch, done thank you! 😅